drivers/storage/vbox/vbox.go

```diff
@@ -27,7 +27,7 @@ func registerConfig() {
 	r.Key(gofig.String, "", volumePath, "", "virtualbox.volumePath")
 	r.Key(gofig.String, "", "", "", "virtualbox.localMachineNameOrId")
 	r.Key(gofig.Bool, "", false, "", "virtualbox.tls")
-	r.Key(gofig.String, "", "SATAController", "", "virtualbox.controllerName")
```

Comment:
Hi @clintonskitson,

I literally just merged this. The comment you made is no longer valid actually as I changed it to "SATA". I forget the original value, but was it "SATA"? If so, we're good.